packages/introspect/src/generate.ts

```diff
@@ -387,25 +387,34 @@ function createWellKnownDefs(): Record<string, JSONSchema> {
             type: 'boolean',
             description: 'A PG bool',
         },
+        'pg_catalog.time': {
+            id: 'pg_catalog.time',
+            type: 'string',
+            format: 'time',
+            description: 'A PG time',
+        },
+        'pg_catalog.timetz': {
+            id: 'pg_catalog.timetz',
+            type: 'string',
+            format: 'time',
+            description: 'A PG timetz',
```

Comment:
Update these comments to explain the format of the generated string.
Explain the caveat with the format being sometimes a little bit different. 
Explain also the caveat that PG dates have a higher precision to JS dates.

Use the venerable `['this','is','a','line'].join('\n')` format for multi string comments.